model/src/test/assets/analyzer-result-with-dependency-graph.yml

```diff
@@ -144,14 +144,6 @@ analyzer:
           url: "ssh://git@github.com/hamcrest/JavaHamcrest.git"
           revision: ""
           path: ""
-      curations:
```

Comment:
I currently don't understand why this is being dropped. Can you help me here?

Reply:
Declared licenses can only be curated with declared licenses mappings now, since then the property is ignored anyway and because it is unrelated to the license graph test I decided to drop it.